script_library/deployment-actions-kvm.sh

```diff
@@ -56,7 +56,8 @@ function deploy_airship(){
 }
 function add_compute(){
     echo "Now Add compute"
-    run_ansible ${socok8s_absolute_dir}/playbooks/add_compute.yml
+    tagged_info=" --tags add_computenode"
+    run_ansible ${socok8s_absolute_dir}/playbooks/add_compute.yml ${tagged_info}
```

Comment:
It is kind of odd to run the specific playbook for add compute and with a specific tag for add compute.  I hope we can come back soon to see if there is possibility of a more elegant solution.

Reply:
> It is kind of odd to run the specific playbook for add compute and with a specific tag for add compute. I hope we can come back soon to see if there is possibility of a more elegant solution.

Yes I agree but not sure where I can add tag because all configure compute and controller is in one role.

Reply:
have you considered to use the deploy playbook with the add_compute_node tag?

Reply:
> have you considered to use the deploy playbook with the add_compute_node tag?

I see add/remove compute is different feature so I dont think we should use deploy playbook for that.  maybe rename deploy playbook to generic name ?